.circleci/config.yml

```diff
@@ -12,10 +12,15 @@ executors:
       - image: metabase/ci:lein-2.9.5-clojure-1.10.3.814
 
   # CircleCI base (Lein 2.9.5) + Node + Headless browsers + Clojure CLI - big one
+  # Maildev runs by default with all Cypress tests
   clojure-and-node-and-browsers:
     working_directory: /home/circleci/metabase/metabase/
     docker:
       - image: circleci/clojure:lein-2.9.5-node-browsers
+      - image: maildev/maildev
+        environment:
+          MB_EMAIL_SMTP_HOST: maildev
```

Comment:
Tbh, not sure if this has any impact (or if it's needed) since we manually set all the parameters later in the test anyway?

Reply:
Either use envvars or set it manually - don't do both - that will just end bad at some point.